Gems/Atom/Feature/Common/Assets/Passes/EditorModeBlur.pass

```diff
@@ -0,0 +1,77 @@
+{
+    // Applies a box blur effect as controlled by the box width
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "PassAsset",
+    "ClassData": {
+        "PassTemplate": {
+            "Name": "EditorModeBlurTemplate",
+            "PassClass": "EditorModeBlurPass",
+            "Slots": [
+                {
+                    "Name": "InputDepth",
+                    "SlotType": "Input",
+                    "ShaderInputName": "m_depth",
+                    "ScopeAttachmentUsage": "Shader"
+                },
+                {
+                    "Name": "InputEntityMask",
+                    "SlotType": "Input",
+                    "ShaderInputName": "m_entityMask",
+                    "ScopeAttachmentUsage": "Shader"
+                },
+                {
+                    "Name": "InputColor",
+                    "SlotType": "Input",
+                    "ShaderInputName": "m_framebuffer",
+                    "ScopeAttachmentUsage": "Shader"
+                },
+                {
+                    "Name": "OutputColor",
+                    "SlotType": "Output",
+                    "ScopeAttachmentUsage": "RenderTarget",
+                    "LoadStoreAction": {
+                        "LoadAction": "DontCare"
+                    }
+                }
+            ],
+            "ImageAttachments": [
+                {
+                    "Name": "OutputAttachment",
+                    "SizeSource": {
+                        "Source": {
+                            "Pass": "This",
+                            "Attachment": "InputColor"
+                        }
+                    },
+                    "FormatSource": {
+                        "Pass": "This",
+                        "Attachment": "InputColor"
+                    }
```

Comment:
Can you make the InputColor an InputOutput slot and write to it directly instead of create a new attachment?

It seems all the new passes create new color attachments. Are they necessary?

Reply:
Also I think it may be possible to merge some passes since they are writing to same output (color output).

Reply:
+1 it looks like several of these passes can be combined

Reply:
@VickyAtAZ @AMZN-alexpete All of the color passes could potentially be combined, but then we would lose the modularity for future use cases and their differing FX (e.g. component mode feedback, selected entity outlining, etc.). We could have one mega pass with all of the discrete pass parameters exposed but you lose a lot of flexibility (you would be locked into a fixed pass sequence, you would not be able to do per-effect depth transitioning, adding new effects for different use cases would be cumbersome, etc.). We still need to do a performance analysis as to the impact of this particular feedback effect for Focus Mode and if the performance between one mega pass and discrete passes is negligible then I would be inclined to not combine them 👍 

@VickyAtAZ I just tried using read/write attachments but to no avail, I'll check with @antonmic if it's possible and whether or not attachments are actually being created for each pass (instead of being recycled) 👍

Reply:
@antonmic The main issue is the final output is a new color render target. This would effect the connections of passes after this pass in the pass tree. And it's a risk for any passes added after this pass.
Also, it's increase peak heap usage because the coexists of input color and new output color.

Reply:
@VickyAtAZ @antonmic I've added tickets LYN-10374 and LYN-10375 to investigate the merging of passes and reuse of existing pass attachments 👍